### DISCUSSION TITLE
Ubuntu 23.10 or Ubuntu 24.04 OS Support

### DISCUSSION THREAD

**Discussion by prakashiOrbit** [pgsty/pigsty#430] [Q&A] [answered]
Hello, I have a general question regarding support for Ubuntu 23.10 and 24.04. When can we expect support for these OS versions? If there won't be support soon, I may need to downgrade my server to Ubuntu 22.04. Thank you.

**Comment by Vonng** [comment, marked as answer]
For ubuntu 23.10, We have no plans to any support non-LTS versions (23.10).
As for Ubuntu 24.04, it is too new, and the packages we need are not fully available on this system.
I guess support will start around the time of 20.04 EOL.

But we always have an enterprise option to customize any OS they want.
